shared/actions/autoreset.tsx

```diff
@@ -1,3 +1,35 @@
-function* autoresetSaga() {}
+import * as Saga from '../util/saga'
+import * as Container from '../util/container'
+
+import * as NotificationsGen from './notifications-gen'
+import * as RouteTreeGen from './route-tree-gen'
+import * as AutoresetGen from './autoreset-gen'
+import * as RPCGen from '../constants/types/rpc-gen'
+
```

Comment:
:fire: blank lines

Reply:
I like them to organize the imports, but I guess we don't use them anywhere else so I can delete them